libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -1,11 +1,12 @@
 import { toFragment } from '../../blocks/global-navigation/utilities/utilities.js';
 import { getConfig } from '../../utils/utils.js'; // TODO: doesn't make sense outside of Milo
-import { replaceKey, replaceText } from '../../features/placeholders.js';
+import { replaceKey, replaceText } from '../placeholders.js';
 
 const CONFIG = {
   selectors: { prompt: '.appPrompt' },
   delay: 7000,
   loaderColor: '#EB1000',
+  icon: '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 133.5 118.1" title="Adobe, Inc."><defs><style>.cls-1 {fill: #eb1000;}</style></defs><g><g><polygon class="cls-1" points="84.1 0 133.5 0 133.5 118.1 84.1 0"/><polygon class="cls-1" points="49.4 0 0 0 0 118.1 49.4 0"/><polygon class="cls-1" points="66.7 43.5 98.2 118.1 77.6 118.1 68.2 94.4 45.2 94.4 66.7 43.5"/></g></g></svg>',
```

Comment:
should we reuse the company icon from the global navigation // or put that icon with the `utilities` which we already consume anyway?

Reply:
I was thinking of that too, but I'd first like to get an understanding of any microbundle limitations. If we need to import a big chunk of code for just this one resource, it might not be worth it for standalone usage. For now I'll just add a comment.